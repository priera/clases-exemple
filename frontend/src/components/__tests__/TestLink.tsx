import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import TestLink from "../TestLink";

describe("this component", () => {
  it("renders correctly", () => {
    const { asFragment, getByText } = render(
      <TestLink page="http://www.facebook.com">Facebook2</TestLink>
    );
    expect(getByText("Facebook2")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
