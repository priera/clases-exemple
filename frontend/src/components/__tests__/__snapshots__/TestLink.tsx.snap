// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this component renders correctly 1`] = `
<DocumentFragment>
  <a
    class="normal"
    href="http://www.facebook.com"
  >
    Facebook2
  </a>
</DocumentFragment>
`;
